test/services/notifications/submit-returned-letter.service.test.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+const { generateUUID, timestampForPostgres, today } = require('../../../app/lib/general.lib.js')
+
+// Test helpers
+const EventHelper = require('../../support/helpers/event.helper.js')
+const NotificationHelper = require('../../support/helpers/notification.helper.js')
+const NotificationModel = require('../../../app/models/notification.model.js')
+
+// Thing under test
+const SubmitReturnedLetterService = require('../../../app/services/notifications/submit-returned-letter.service.js')
+
+describe('Submit Returned Letter Service', () => {
+  let eventId
+  let notifyId
+  const todaysDate = today()
```

Comment:
Our convention is to put consts above lets. Generally we leave a gap between them as well

```suggestion
  const todaysDate = today()

  let eventId
  let notifyId
```